Misc/NEWS.d/next/Core and Builtins/2024-04-11-21-17-23.gh-issue-117431.ZxdAFN.rst

```diff
@@ -0,0 +1 @@
+Improve performance of :func:`str.startswith`, :func:`str.endswith`, :func:`str.removeprefix` and :func:`str.removesuffix`.
```

Comment:
I think it is more accurate to mark these up as methods, but I don't think it matters when they are rendered.

```suggestion
Improve performance of :meth:`str.startswith`, :meth:`str.endswith`, :meth:`str.removeprefix` and :meth:`str.removesuffix`.
```